src/overseer/store/jdbc.clj

```diff
@@ -0,0 +1,238 @@
+(ns ^:no-doc overseer.store.jdbc
+  "Implementation of overseer.core/Store for JDBC"
+  (:require [clojure.java.jdbc :as j]
+            [clojure.set :as set]
+            [clojure.string :as string]
+            [framed.std.core :as std]
+            (loom graph)
+            (overseer
+              [core :as core]
+              [util :as util])))
+
+(defn- in-placeholders
+  "Generate the appropriate number of `?` placeholders for an IN clause
+  for values `vs`"
+  [vs]
+  (string/join "," (repeat (count vs) "?")))
+
+(defn where-map->clause
+  "Given a map of {keyword-column-name value}, format it for use in a SQL WHERE
+  clause
+
+  Ex:
+    (where-map->clause {:age 30 :email \"bob@example.com\"})
+    ; => \"age = 30 AND email = 'bob@example.com'\""
+  [where-map]
+  (let [format-value
+        (fn [v] (if (string? v) (format "'%s'" v) v))]
+    (->> where-map
+         (map (fn [[k v]] (str (name k) " = " (format-value v))))
+         (string/join " AND "))))
+
+(defn job->jdbc-map [job]
+  {:pre [job]}
+  (let [{:keys [job/id job/type job/args job/status
+                job/failure job/heartbeat]} job]
+    (-> {:id id
+         :type (name type)
+         :status (name status)}
+        (std/when-assoc :args (some-> args std/to-edn))
+        (std/when-assoc :failure (some-> failure std/to-edn))
+        (std/when-assoc :heartbeat heartbeat))))
+
+(defn jdbc-map->job [jdbc-map]
+  {:pre [jdbc-map]}
+  (-> (std/map-kv #(keyword "job" (name %)) identity jdbc-map)
+      (util/when-update :job/status keyword)
```

Comment:
I used jdbc-map to refer to the 'raw' result map you get, i.e. `{:id, :status, :args EDN string}`, which needs to be parsed into a "Job". I believe when-update was applied overly generously for status/type, fixing that